### ISSUE TITLE
preserve pitch when changing playbackRate

### ISSUE THREAD

**Issue by Aparus** [katspaugh/wavesurfer.js#1710]
I studied this topic  [feature: soundtouch.js filter to preserve pitch when changing tempo](https://github.com/katspaugh/wavesurfer.js/pull/1214 ) 
and other related issues, and what I understand, first it was utils/kali.js , then soundtouch.js - exact in the the wevesurfer.js/webAudio code. 

But then suddenly them disappeared from ws code, and appeared in example [Stretcher](https://github.com/katspaugh/wavesurfer.js/tree/master/example/stretcher) 

I suppose this is an old approach, without modules and imports (just using window.soundtouch). 

Does somebody know, how to preserve peach in 2019 while playbackRate changed? 
I mean module based Wavesurfer v3 .  
 
### Wavesurfer.js version(s):
^3.0.0

### Browser and operating system version(s):
Chrome 75.0.3770.142 (Official Build) (64-bit)
Ubuntu 19.04

### Code needed to reproduce the issue:

https://codepen.io/aparus/pen/pMwadr

**Comment by Aparus** [comment]
Finally I used code from [Stretcher example](https://github.com/katspaugh/wavesurfer.js/tree/master/example/stretcher) - thanks to its author!
I putted whole code from `soundtouch.js` to module, where just removed wrapper 
`(function(window) {.... } ) ` and at the end of file changed 
`window.soundtouch = { .... } ` 
to 
```
export default {
  RateTransposer: RateTransposer,
  Stretch: Stretch,
  SimpleFilter: SimpleFilter,
  SoundTouch: SoundTouch,
  WebAudioBufferSource: WebAudioBufferSource,
  getWebAudioNode: getWebAudioNode
}
```
Then in module when I initialize wavesurfer I imported  `import soundtouch from './soundtouchFilter'
`
 and copied all code from `wavesurfer.on('ready', function() { ...} )` from example, and just changed `window.soundtouch` to `soundtouch`

If someone are interested, here is [my code](https://github.com/Aparus/frazish/tree/master/src/wavesurfer) . 

In the web people are saying, than Soundtouch filter is better than others. Yes, It is better than nothing. But for example in youtube when you change playback rate even to 0.5, sound is good. With this filter I hear echo. If someone will find some better filter or solution, please notice somehow.

**Comment by Caixz** [comment]
Can this problem be solved? When the audio frequency is 0.5 or 2 times, the original sound will disappear and the female will become male. But I think the multiples provided by the original label < audio / > can remain normal
